Builds/VisualStudio2015/RippleD.vcxproj

```diff
@@ -197,7 +197,7 @@
     <ClCompile>
       <PreprocessorDefinitions>HAVE_USLEEP=1;SOCI_CXX_C11=1;_WIN32_WINNT=0x6000;BOOST_NO_AUTO_PTR;DEPRECATED_IN_MAC_OS_X_VERSION_10_7_AND_LATER;NDEBUG;NOMINMAX;OPENSSL_NO_SSL2;WIN32_CONSOLE;_CRT_SECURE_NO_WARNINGS;_SCL_SECURE_NO_WARNINGS;_SILENCE_STDEXT_HASH_DEPRECATION_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <AdditionalIncludeDirectories>..\..\build\proto;..\..\src;..\..\src\beast;..\..\src\beast\extras;..\..\src\beast\include;..\..\src\protobuf\src;..\..\src\protobuf\vsprojects;..\..\src\soci\include;..\..\src\soci\src;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <DisableSpecificWarnings>4800;4244;4267;4018</DisableSpecificWarnings>
+      <DisableSpecificWarnings>4800;4244;4267;4503;4018</DisableSpecificWarnings>

```

Comment:
@mellery451 I add `/wd` switches to `add_compile_options` to disable these warnings in the CMake generated visual studio project (see https://github.com/seelabs/rippled/blob/4511ab8d69c1e8b9f7f38a4fa236bcd2afbe1f22/CMakeLists.txt#L554-L558) Are you saying those don't work?